forge/testdata/DebugLogsTest.sol

```diff
@@ -24,6 +24,11 @@ contract DebugLogsTest is DSTest {
         emit log("three");
         fails.failure();
     }
+
+    function testFailWithRequire() public {
+        emit log("four");
+        require(false);
```

Comment:
I think priorly failures from `require` where still emitting but `revert` were not